src/Fixer/FunctionNotation/NoUnreachableDefaultArgumentValueFixer.php

```diff
@@ -53,18 +48,38 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     public function getDefinition()
     {
         return new FixerDefinition(
-            'In method arguments there must not be arguments with default values before non-default ones.',
+            'In function and method arguments there must not be arguments with default values before non-default ones.',
             array(
                 new CodeSample(
                     '<?php
 function example($foo = "two words", $bar) {}
 '
                 ),
-            )
+            ),
+            null,
+            null,
+            null,
+            'Modifies the signature of methods/functions, risky when using systems relying on those (for example through reflection).'
```

Comment:
could we ref. to example from Symfony, as the statement is not obvious for all devs.